Fix version_generator.py to also work with Python3
atch created by Simon Branford of the BEAR Software team at the University of Birmingham
--- pgm/version_generator.py.orig	2020-04-24 08:35:18.201480000 +0100
+++ pgm/version_generator.py	2020-04-24 08:36:08.643351000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
 
 import os
 import platform
@@ -6,9 +6,9 @@
 
 build_date = time.strftime ("%Y-%m-%d")
 build_time = time.strftime ("%H:%M:%S")
-build_rev = filter (str.isdigit, "$Revision: 1487 $")
+build_rev = "".join(list(filter (str.isdigit, "$Revision: 1487 $")))
 
-print """
+print("""
 /* vim:ts=8:sts=8:sw=4:noai:noexpandtab
  * 
  * OpenPGM version.
@@ -50,6 +50,6 @@
 
 
 /* eof */
-"""%(build_date, build_time, platform.system(), platform.machine(), build_rev)
+"""%(build_date, build_time, platform.system(), platform.machine(), build_rev))
 
 # end of file
